assets/javascripts/admin_user.js

```diff
@@ -37,4 +37,23 @@ function setup_admin_user() {
             }
         });
     });
+
+    window.deleteUser = function(id){
```

Comment:
Sorry but these missing spaces just sting into my eye. I've already provided you a regex substitution to fix these automatically in another review.